### ISSUE TITLE
Unsupported installation arguments

### ISSUE THREAD

**Issue by BastiaanLouis** [dotnetcore-chocolatey/dotnetcore-chocolateypackages#49]
Installation is done using some default arguments. This is also mentioned in the nuspec:
```
The native installer is instructed to skip installing the .NET Core Runtime (`OPT_INSTALL_LTS_REDIST=0 OPT_INSTALL_FTS_REDIST=0`).
```
However, the underlying installation does not seem to support these arguments. [The documentation](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1#install-the-net-core-hosting-bundle) says something about OPT_NO_RUNTIME=1 to skip  installing the runtime and OPT_NO_SHAREDFX=1 to skip installing the shared framework.

So my question: Am I doing something wrong or do we need a change to the default arguments within the nuspec?

**Comment by BastiaanLouis** [comment]
Seems to be kind of a duplicate with #13 but that one is mentioning 2.0 and is open for a long time.
@riezebosch @jberezanski what do you think about this issue? Am I expecting the wrong thing (and is the description within the nuspec incorrect), are the used arguments deprecated and is this an issue indeed or ...... something else.... ?

**Comment by riezebosch** [comment]
I think you're right and the command line options have changed over time

**Comment by jberezanski** [comment]
Yes, the package install script stopped being updated around 2.0. Besides, the current architecture of having only a single package for all versions and using AU streams to produce updated packages for each major.minor version family makes it rather inconvenient to differentiate behavior between the version families. I'm hoping to change it as part of #45.
